Protos/generate.sh

```diff
@@ -201,6 +201,26 @@ function generate_service_messages_interop_tests {
   done
 }
 
+function generate_qps_code {
+  local protos=(
+    "$here/upstream/grpc/testing/payloads.proto"
+    "$here/upstream/grpc/testing/control.proto"
+    "$here/upstream/grpc/testing/messages.proto"
+    "$here/upstream/grpc/testing/stats.proto"
+    "$here/upstream/grpc/testing/benchmark_service.proto"
+    "$here/upstream/grpc/testing/worker_service.proto"
+  )
+
+  local output="$root/Sources/performance-worker/Generated"
+  
+  generate_message "$here/upstream/grpc/core/stats.proto" "$here/upstream" "$output" "Visibility=Internal" "FileNaming=PathToUnderscores"
```

Comment:
here we use `"FileNaming=PathToUnderscores"` but below we use `"FileNaming=DropPath"` – why?

Reply:
There are 2 stat.proto file - one in grpc/core and one in grpc/testing. As all the other .proto files are located within 'grpc/testing' I was thinking we can drop the path from their names, and keep the path only for the 'stats.proto' file from the other module, to differentiate the 2 files containing generated code for the 2 'stats.proto' files.

Reply:
Oh, I didn't realise there were two, thanks for explaining. In that case I'd rather we stick to one naming style if possible.